### PR TITLE
cloud: remove beta from serverless docs

### PR DESCRIPTION
**Pull request by ran-huang**
### What is changed, added or deleted? (Required)

remove beta from serverless docs

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v7.2 (TiDB 7.2 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### CODE REVIEW

File: tidb-cloud/serverless-faqs.md [hunk 1]
```diff
@@ -20,17 +20,15 @@ TiDB Serverless offers the TiDB database with full HTAP capabilities for you and
...
+<!--To be confirmed-->
+Each TiDB Serverless cluster has a free [usage quota](/tidb-cloud/serverless-limitations.md#usage-quota). Usage beyond the free quota will be charged. Once the free quota of a cluster is reached, the read and write operations on this cluster will be throttled until you [increase the quota](/tidb-cloud/manage-serverless-spend-limit.md#update-spend-limit) or the usage is reset upon the start of a new month.
 
 For more information, see [TiDB Serverless usage quota](/tidb-cloud/select-cluster-tier.md#usage-quota).
 
-### What does it mean for beta release?
-
-TiDB Serverless is in beta while we continuously add new features and improve existing features before it becomes generally available. We do not provide SLA for beta products. Therefore, TiDB Serverless should **NOT** be used in production currently.
```
Comment (ran-huang) [inline tidb-cloud/serverless-faqs.md:31]: Can we safely delete this faq?

File: tidb-cloud/serverless-limitations.md [hunk 2]
```diff
@@ -26,7 +26,7 @@ We are constantly filling in the feature gaps between TiDB Serverless and TiDB D
 
 ### Transaction
 
-- The total size of a single transaction is set to no more than 10 MB on TiDB Serverless during the beta phase.
+- The total size of a single transaction is set to no more than 10 MB on TiDB Serverless during the beta phase. <!--To be confirmed-->
```
Comment (ran-huang) [inline tidb-cloud/serverless-limitations.md:29]: Is this limitation still applicable to GA?

File: develop/dev-guide-aws-appflow-integration.md [hunk 3]
```diff
@@ -244,5 +244,5 @@ test> SELECT * FROM sf_account;
 
 - If anything goes wrong, you can navigate to the [CloudWatch](https://console.aws.amazon.com/cloudwatch/home) page on the AWS Management Console to get logs.
 - The steps in this document are based on [Building custom connectors using the Amazon AppFlow Custom Connector SDK](https://aws.amazon.com/blogs/compute/building-custom-connectors-using-the-amazon-appflow-custom-connector-sdk/).
-- [TiDB Serverless](https://docs.pingcap.com/tidbcloud/select-cluster-tier#tidb-serverless-beta) is **NOT** a production environment.
+- [TiDB Serverless](https://docs.pingcap.com/tidbcloud/select-cluster-tier#tidb-serverless) is **NOT** a production environment.
```
Comment (shizn) [inline develop/dev-guide-aws-appflow-integration.md:247]: GA means now it can be a production env.

File: tidb-cloud/select-cluster-tier.md [hunk 4]
```diff
@@ -64,8 +65,8 @@ To get the prefix for your cluster, take the following steps:
 
 ### TiDB Serverless special terms and conditions
 
-- TiDB Serverless is currently in beta and does not have uptime SLA guarantee during beta phase. If you use TiDB Serverless beta to store a commercial or production dataset, any potential risk associated with the use should be taken on your own, and PingCAP shall not be liable for any damage.
-- Some of TiDB Cloud features are partially supported or not supported on TiDB Serverless. See [TiDB Serverless Limitations](/tidb-cloud/serverless-limitations.md) for details.
+<!--To be confirmed-->
+Some of TiDB Cloud features are partially supported or not supported on TiDB Serverless. See [TiDB Serverless Limitations](/tidb-cloud/serverless-limitations.md) for details.
```
Comment (shizn) [inline tidb-cloud/select-cluster-tier.md:69]: LGTM.

File: tidb-cloud/serverless-limitations.md (hunk 2 above)
Comment (shizn) [reply tidb-cloud/serverless-limitations.md:29]: This limitation doc will be updated by xiaoguang in a separate PR.

File: develop/dev-guide-aws-appflow-integration.md (hunk 3 above)
Comment (ran-huang) [inline develop/dev-guide-aws-appflow-integration.md:247]:
```suggestion
```

Comment (ran-huang) [reply develop/dev-guide-aws-appflow-integration.md:247]: I'll delete this line.

File: tidb-cloud/select-cluster-tier.md [hunk 5]
```diff
@@ -10,12 +10,13 @@ The cluster tier determines the throughput and performance of your cluster.
...
-- [TiDB Serverless](#tidb-serverless-beta)
+- [TiDB Serverless](#tidb-serverless)
 - [TiDB Dedicated](#tidb-dedicated)
 
-## TiDB Serverless (Beta)
+## TiDB Serverless
 
-TiDB Serverless (previously called Developer Tier) is a fully managed service of TiDB. It's still in beta and should not be used in production. However, you can use TiDB Serverless clusters for non-production workloads such as prototype applications, hackathons, academic courses, or to provide a temporary data service for your datasets.
+<!--To be confirmed-->
+TiDB Serverless (previously called Developer Tier) is a fully managed service of TiDB. You can use TiDB Serverless clusters for non-production workloads such as prototype applications, hackathons, academic courses, or to provide a temporary data service for your datasets.
```
Comment (ran-huang) [inline tidb-cloud/select-cluster-tier.md:19]: @shizn Do we need to update the workload examples here?
```suggestion
TiDB Serverless (previously called Developer Tier) is a fully managed service of TiDB. You can use TiDB Serverless clusters for production workloads such as prototype applications, hackathons, academic courses, or to provide a temporary data service for your datasets.
```

File: tidb-cloud/select-cluster-tier.md [hunk 6]
```diff
@@ -64,8 +65,8 @@ To get the prefix for your cluster, take the following steps:
 
 ### TiDB Serverless special terms and conditions
 
-- TiDB Serverless is currently in beta and does not have uptime SLA guarantee during beta phase. If you use TiDB Serverless beta to store a commercial or production dataset, any potential risk associated with the use should be taken on your own, and PingCAP shall not be liable for any damage.
-- Some of TiDB Cloud features are partially supported or not supported on TiDB Serverless. See [TiDB Serverless Limitations](/tidb-cloud/serverless-limitations.md) for details.
+<!--To be confirmed-->
```
Comment (ran-huang) [inline tidb-cloud/select-cluster-tier.md:68]:
```suggestion
```

File: tidb-cloud/serverless-limitations.md (hunk 2 above)
Comment (ran-huang) [inline tidb-cloud/serverless-limitations.md:29]:
```suggestion
- The total size of a single transaction is set to no more than 10 MB on TiDB Serverless during the beta phase.
```

**Review by shizn — APPROVED** [review]

**Review by hfxsd — APPROVED** [review]

**Pull request merged**